opba-protocols/hbci-protocol/src/main/java/de/adorsys/opba/protocol/hbci/entrypoint/pis/HbciGetPaymentInfoEntrypoint.java

```diff
@@ -0,0 +1,60 @@
+package de.adorsys.opba.protocol.hbci.entrypoint.pis;
+
+import com.google.common.collect.ImmutableMap;
+import de.adorsys.opba.protocol.api.dto.ValidationIssue;
+import de.adorsys.opba.protocol.api.dto.context.ServiceContext;
+import de.adorsys.opba.protocol.api.dto.request.payments.PaymentInfoBody;
+import de.adorsys.opba.protocol.api.dto.request.payments.PaymentInfoRequest;
+import de.adorsys.opba.protocol.api.dto.result.body.ValidationError;
+import de.adorsys.opba.protocol.api.dto.result.fromprotocol.Result;
+import de.adorsys.opba.protocol.api.pis.GetPaymentInfoState;
+import de.adorsys.opba.protocol.bpmnshared.dto.DtoMapper;
+import de.adorsys.opba.protocol.bpmnshared.service.eventbus.ProcessEventHandlerRegistrar;
+import de.adorsys.opba.protocol.hbci.entrypoint.HbciOutcomeMapper;
+import de.adorsys.opba.protocol.hbci.entrypoint.HbciResultBodyExtractor;
+import lombok.RequiredArgsConstructor;
+import org.flowable.engine.RuntimeService;
+import org.flowable.engine.runtime.ProcessInstance;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static de.adorsys.opba.protocol.api.common.ProtocolAction.GET_PAYMENT_INFORMATION;
+import static de.adorsys.opba.protocol.bpmnshared.GlobalConst.CONTEXT;
+import static de.adorsys.opba.protocol.hbci.constant.GlobalConst.HBCI_REQUEST_SAGA;
+
+/**
+ * Entry point to get hbci payment status.
+ */
+@Service("hbciGetPaymentInfoState")
+@RequiredArgsConstructor
+public class HbciGetPaymentInfoEntrypoint implements GetPaymentInfoState {
```

Comment:
Since this class duplicates HbciGetPaymentStatusEntrypoint, can we create single Base abstract class and make HbciGetPaymentInfoEntrypoint, HbciGetPaymentStatusEntrypoint extend it for code reuse?